CMakeLists.txt

```diff
@@ -137,30 +135,22 @@ if (ENABLE_ALL_DEBUG_FACILITIES)
 endif()
 
 if (ENABLE_ALL_THE_DEBUG_MACROS)
-    include(${CMAKE_SOURCE_DIR}/Meta/CMake/all_the_debug_macros.cmake)
+    include("${SerenityOS_SOURCE_DIR}/Meta/CMake/all_the_debug_macros.cmake")
 endif(ENABLE_ALL_THE_DEBUG_MACROS)
 
 configure_file(AK/Debug.h.in AK/Debug.h @ONLY)
 configure_file(Kernel/Debug.h.in Kernel/Debug.h @ONLY)
 
+include(Meta/CMake/wasm_spec_tests.cmake)
+
 include_directories(Userland/Libraries)
 include_directories(.)
 include_directories(${CMAKE_BINARY_DIR})
 
-set(write_if_different ${CMAKE_SOURCE_DIR}/Meta/write-only-on-difference.sh)
+set(write_if_different ${SerenityOS_SOURCE_DIR}/Meta/write-only-on-difference.sh)
```

Comment:
This script was deleted. This variable should be removed as well.

Reply:
The variable is deleted after the commit that makes this change, in the last commit of the PR (after all callers are refactored)
The commit that deletes the script removes the variable.

Reply:
It's changed in https://github.com/SerenityOS/serenity/pull/9281/commits/af2c78c3df835bbb644285c7a82596f8ca7efdb7 but missing from https://github.com/SerenityOS/serenity/pull/9281/commits/d15f00066fd814f702c509590ce9a71a0b8c99d7

Reply:
All right. I've modified the history so that the only commit that touches this variable is the one that deletes it.